### DISCUSSION TITLE
Notifications badge never goes away, despite having no notifications.

### DISCUSSION THREAD

**Discussion by Qix-** [community/community#113459] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Title. Been happening for a few weeks now. The notification badge on the inbox icon in the top right seems to be stuck, despite having no notifications. Even when I get them and mark them as completed again and go back to inbox zero, the badge remains.

**Comment by Limess** [comment, marked as answer]
This seems to be the same as this bug:

https://github.com/orgs/community/discussions/111217

FYI I fixed it for myself by marking all notifications as read via the API, in my case I used the Github CLI:

```cli
gh api --method PUT -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /notifications -F read=true
```

**Reply by Qix-** [reply]
Finally got around to testing this, it worked 😌 Thank you very much!

**Reply by 3zcurdia** [reply]
It worked for me, thank you very much! 🙇‍♂️
